securedrop/source_templates/base.html

```diff
@@ -33,7 +33,7 @@
         <a href="{{ url_for('main.logout') }}" class="sd-button btn pull-right" id="logout">{{ gettext('LOG OUT') }}</a>
       {% endif %}
         <hr class="no-line">
-
+        {% include 'flashed.html' %}
```

Comment:
This will cause flashed messages to be flashed twice:

<img width="981" alt="screen shot 2017-10-25 at 11 04 09 am" src="https://user-images.githubusercontent.com/7832803/32014907-4977bd38-b974-11e7-8c96-fedb58192aa2.png">